README.md

```diff
@@ -27,10 +27,17 @@ First, add LeafErrorMiddleware as a dependency in your `Package.swift` file:
 ```swift
 dependencies: [
     // ...,
-    .package(name: "LeafErrorMiddleware", url: "https://github.com/brokenhandsio/leaf-error-middleware.git", from: "3.0.0")
```

Comment:
This needs to stay, the name of the package is `LeafErrorMiddleware` - because the name of the repo is `leaf-error-middleware` you'll get a SwiftPM error if you do this

Reply:
That's a deprecated way of describing a package. I have a project that looks like this that is compiling error free right now
```swift
dependencies: [
        // 💧 A server-side Swift web framework.
        ...
        .package(url: "https://github.com/brokenhandsio/leaf-error-middleware.git", from: "4.0.0")
        
    ],
    targets: [
        .target(
            name: "App",
            dependencies: [
               ...
                .product(name: "Vapor", package: "vapor"),
                .product(name: "LeafErrorMiddleware", package: "leaf-error-middleware")
            ],
            ...
		),
		...
    ]
```

Reply:
<img width="1443" alt="Screen Shot 2022-06-28 at 1 51 06 AM" src="https://user-images.githubusercontent.com/59743986/176124209-c8180d5f-41ba-47b3-9adb-ccf28d356f87.png">

Reply:
Huh, this has changed in 5.6. Another breaking change in a minor release in Swift 😒